packages/contracts/build/contracts/ProxyFactory_solc.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
The other ABI files in this directory do not have the _solc suffix in their filenames.
Is it intentional that ProxyFactory_solc.json and IdentityProxy_solc.json do ?

Reply:
It's because Truffle can only support either version 4 or 5 of Solidity but not both, and the new Proxy contract requires version 5. I've appended _solc to the contract names so we know they were compiled with Solc instead of Truffle.

Reply:
Got it. Thanks for explaining.
Probably worth adding a README.md to explain since I'm sure we'll forgot why it is the case and ask again... Same thing for the .s files.